crates/bevy_internal/src/lib.rs

```diff
@@ -193,3 +190,40 @@ pub mod dynamic_plugin {
     //! Dynamic linking of plugins
     pub use bevy_dynamic_plugin::*;
 }
+
+/// `use bevy::prelude::*;` to import common components, bundles, and plugins.
+pub mod prelude {
+    #[doc(hidden)]
+    pub use crate::{
+        app::prelude::*, asset::prelude::*, core::prelude::*, core_pipeline::prelude::*,
+        ecs::prelude::*, hierarchy::prelude::*, input::prelude::*, log::prelude::*,
+        math::prelude::*, render::prelude::*, text::prelude::*, time::prelude::*,
+        transform::prelude::*, ui::prelude::*, utils::prelude::*, window::prelude::*,
```

Comment:
I believe some of them should still be gated by feature.

Reply:
I do not know what that means. What do you want me to do to fix it?

Reply:
Bevy is modular, so you can easily swap some crates with your own (for example you might have your own UI crate you want to use).

So some of those should be behind a feature flag e.g.:
```rust
    #[cfg(feature = "bevy_ui")]
    pub use crate::ui::prelude::*;
```

You can take a look at [prelude.rs](https://github.com/bevyengine/bevy/blob/main/crates/bevy_internal/src/prelude.rs) to see which should be behind a feature, or at [Cargo.toml](https://github.com/bevyengine/bevy/blob/main/crates/bevy_internal/Cargo.toml#L180-L195). 

Also, you should get compile errors when running:
```bash
cargo build --no-default-feature
```